### PR TITLE
Mainnet staging: fixes

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Explanation of the solution
- platform admin links not showing - fix hide logic

### Others
- cherry-pick fixes from other working branch

### TODO
- cherry-pick this back to `RC-v1.8`

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function AdminLinks(props: WalletState) {
   const isLoading =
     isApLoading || isApFetching || isReviewLoading || isReviewFetching;
 
-  if (isLoading || !isApMember || !isReviewMember) return null;
+  if (isLoading || (!isApMember && !isReviewMember)) return null;
```
Comment (ap-justin) [inline src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx:28]: should show even single
image

File: src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Beneficiary.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ export default function Beneficiary({ classes = "" }) {
                 <span
                   className={`${
                     checked ? "bg-blue/20" : "bg-blue/5"
-                  } text-gray-d2 @sm:w-36 cursor-pointer px-2 py-1 text-center text-xs uppercase`}
+                  } text-gray-d2 dark:text-white @sm:w-36 cursor-pointer px-2 py-1 text-center text-xs uppercase`}
```
Comment (ap-justin) [inline src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Beneficiary.tsx:58]: tab color
image

File: src/pages/Admin/Charity/index.tsx [hunk 3]
```diff
@@ -57,7 +57,16 @@ const LINK_GROUPS: { [key in EndowmentType]: LinkGroup[] } = {
     },
   ],
   ast: [
-    ...COMMON,
+    {
```
Comment (ap-justin) [inline src/pages/Admin/Charity/index.tsx:60]: image

deposit should be shown in ast admin

File: src/pages/Admin/Context.tsx [hunk 4]
```diff
@@ -159,14 +159,20 @@ export const useAdminContext = <T extends AdminType = any>(
       switch (op) {
         case "withdraw-liquid":
           // if beneficiary is endowment, sender must be multisig
-          if (closed && closingBeneficiary.type === "wallet") {
-            return closingBeneficiary.value === sender;
+          if (closed) {
```
Comment (ap-justin) [inline src/pages/Admin/Context.tsx:162]: `isDelegated` should be `false` when bene type is endowment

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**